sbc_translation/models/translation_competence.py

```diff
@@ -16,7 +16,7 @@ class TranslationCompetence(models.Model):
         "translation.competence", "Fallback competence",
         help="Letters will move to this pool if they sit for too long waiting to be translated."
     )
-    name = fields.Char(compute="_compute_name")
+    name = fields.Char(compute="_compute_name", store=True)  # We need to store it to filter on it
```

Comment:
You should add @api.depends decorator on the compute method to retrigger the compute if source or dest language is changed. That should not happen often, but we prefer making sure the name is always correct.